projects/addon-doc/src/components/language-switcher/language-switcher.component.html

```diff
@@ -0,0 +1,26 @@
+<tui-select
+    tuiTextfieldSize="m"
+    [valueContent]="content"
+    [formControl]="language"
+>
+    <ng-content></ng-content>
+    <ng-template tuiDataList>
+        <tui-data-list>
+            <button
+                *ngFor="let name of names"
+                tuiOption
+                [value]="name"
+                (click)="switcher.set(name)"
+            >
+                {{ flags.get(name) }}
+            </button>
+        </tui-data-list>
+    </ng-template>
+</tui-select>
+
+<ng-template
+    #content
+    let-name
+>
+    {{ name }}
```

Comment:
This doesn't do anything and can be removed, it is basically `item.toString()` as content, which is the default.

Reply:
fixed